app/services/live_lookup/solr.rb

```diff
@@ -90,20 +87,11 @@ def home_location_code
       end
 
       def current_location_code
-        item['current_location']
-      end
-
-      def valid_current_location?
-        return false if current_location_code.blank? ||
-                        current_location_code == 'CHECKEDOUT' ||
-                        current_location_same_as_home_location?
-
-        true
+        @current_location_code ||= item['current_location']
       end
 
-      def current_location_same_as_home_location?
-        current_location.name == home_location.name ||
-          Constants::CURRENT_HOME_LOCS.include?(current_location_code)
+      def location_lookup
```

Comment:
What is the purpose of this method? I'm not seeing anything calling it.

Reply:
Looks like it is a function I missed when removing old code. Thanks for the catch.